src/user.rs

```diff
@@ -0,0 +1,46 @@
+use std::fs::File;
+use std::io::{Read, Write};
+use std::path::{Path, PathBuf};
+
+use rustc_serialize::json;
+
+use project_types::Project;
+use Error;
+
+
+pub fn new_user<P: AsRef<Path>>(project: String, public: P, name: String) -> Result<(), Error> {
+
+	// Get public key from file
+	let pub_key = try!(file_as_string(public));
+
+	// Get existing Protonfile
+    let mut path = PathBuf::from(Path::new(&project));
+    path.push("Protonfile.json");
+    let protonfile = try!(file_as_string(&path));
+
+    // Load Project from existing file
+   	let mut project: Project = try!(json::decode(&protonfile).map_err(Error::JsonDecode));
+
+   	// Add user
+   	project.add_user(name, pub_key);
+
+   	// Save file
+   	let pretty_json = json::as_pretty_json(&project);
```

Comment:
This as well, turn it into a `fn write_protonfile(&Project) -> Result<(), Error>` and toss it in Utils. Don't change the code in Init module to use it, because this will assume the Protonfile goes in the current directory.